numpy/random/mtrand/mtrand.pyx

```diff
@@ -1224,7 +1224,9 @@ cdef class RandomState:
         Draw samples from a uniform distribution.
 
         Samples are uniformly distributed over the half-open interval
-        ``[low, high)`` (includes low, but excludes high).  In other words,
+        ``[low, high)`` . The end point ``high`` may or may not be included
+        in the range depending on the floating-point rounding in the equation
+        ``(b-a) * random_sample() + a``.  In other words,
```

Comment:
What are `a` and `b`? They are not arguments to `uniform`.

Reply:
Those were meant to be `low` and `high` respectively. I mistakenly took it directly from the `numpy.random.random_sample()` docs. Correcting.